invenio_files_rest/storage.py

```diff
@@ -127,3 +132,83 @@ def send_file(self, restricted=False):
                 content_md5=self.file.checksum)
         except Exception as e:
             raise StorageError('Could not send file: {}'.format(e))
+
+
+class AmazonS3(Storage):
+    def __init__(self, object, fileinstance):
+        super(AmazonS3, self).__init__(object, fileinstance)
+        if "AWS_KEY" in current_app.config:
```

Comment:
I didn't because I was thinking that perhaps AWS_SECRET and AWS_KEY could be invenio wide, but I will do rename them.